pkg/acsengine/engine.go

```diff
@@ -1564,6 +1564,10 @@ func (t *TemplateGenerator) getTemplateFuncMap(cs *api.ContainerService) templat
 		"EnableDataEncryptionAtRest": func() bool {
 			return helpers.IsTrueBoolPointer(cs.Properties.OrchestratorProfile.KubernetesConfig.EnableDataEncryptionAtRest)
 		},
+		"EnablePodSecurityPolicy": func() bool {
```

Comment:
It doesn't look like this function is ever used.

Reply:
removed, thanks !